docker-compose.dev-thirdparty.yml

```diff
@@ -0,0 +1,429 @@
+      - IN_DOCKER=True
+      - AIRBYTE_ENABLED=True
+      - FLASK_APP=${FLASK_APP}
+      - FLASK_ENV=${FLASK_ENV}
+      - FLASK_DEBUG=${FLASK_DEBUG}
+      - FLASK_RUN_PORT=${FLASK_RUN_PORT}
+      - SECRET_KEY=${SECRET_KEY}
+      - SEND_FILE_MAX_AGE_DEFAULT=${SEND_FILE_MAX_AGE_DEFAULT}
+      - DB_HOST=${DB_HOST}
+      - DB_USERNAME=${DB_USERNAME}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_PORT=${DB_PORT}
+      - META_DATABASE=${META_DATABASE}
+      - DATA_DATABASE=${DATA_DATABASE}
+      - DATABASE_URL_CG_DB=${DATABASE_URL_CG_DB}
+      - INTEGRATION_SERVER=${INTEGRATION_SERVER}
+      - INTEGRATION_DB_HOST=${INTEGRATION_DB_HOST}
+      - INTEGRATION_DB_USERNAME=${INTEGRATION_DB_USERNAME}
+      - INTEGRATION_DB_PASSWORD=${INTEGRATION_DB_PASSWORD}
+      - INTEGRATION_DB_PORT=${INTEGRATION_DB_PORT}
+      - INTEGRATION_DATABASE=${INTEGRATION_DATABASE}
+      - CELERY_RESULT_BACKEND=${CELERY_RESULT_BACKEND}
+      - CELERY_BROKER_URL=${CELERY_BROKER_URL}
+      - CACHE_DEFAULT_TIMEOUT=${CACHE_DEFAULT_TIMEOUT}
+      - MULTIDIM_ANALYSIS_FOR_ANOMALY=${MULTIDIM_ANALYSIS_FOR_ANOMALY}
+      - MAX_SUBDIM_CARDINALITY=${MAX_SUBDIM_CARDINALITY}
+      - TOP_DIMENSIONS_FOR_ANOMALY_DRILLDOWN=${TOP_DIMENSIONS_FOR_ANOMALY_DRILLDOWN}
+      - MIN_DATA_IN_SUBGROUP=${MIN_DATA_IN_SUBGROUP}
+      - TOP_SUBDIMENSIONS_FOR_ANOMALY=${TOP_SUBDIMENSIONS_FOR_ANOMALY}
+      - MAX_ROWS_FOR_DEEPDRILLS=${MAX_ROWS_FOR_DEEPDRILLS}
+      - MAX_FILTER_SUBGROUPS_ANOMALY=${MAX_FILTER_SUBGROUPS_ANOMALY}
+      - MAX_DEEPDRILLS_SLACK_DAYS=${MAX_DEEPDRILLS_SLACK_DAYS}
+      - MAX_ANOMALY_SLACK_DAYS=${MAX_ANOMALY_SLACK_DAYS}
+      - DAYS_OFFSET_FOR_ANALTYICS=${DAYS_OFFSET_FOR_ANALTYICS}
+      - HOURS_OFFSET_FOR_ANALTYICS=${HOURS_OFFSET_FOR_ANALTYICS}
+      - DEEPDRILLS_HTABLE_MAX_PARENTS=${DEEPDRILLS_HTABLE_MAX_PARENTS}
+      - DEEPDRILLS_HTABLE_MAX_CHILDREN=${DEEPDRILLS_HTABLE_MAX_CHILDREN}
+      - DEEPDRILLS_HTABLE_MAX_DEPTH=${DEEPDRILLS_HTABLE_MAX_DEPTH}
+      - DEEPDRILLS_ENABLED_TIME_RANGES=${DEEPDRILLS_ENABLED_TIME_RANGES}
+      - TIMEZONE=${TIMEZONE}
+      - REACT_APP_EVENT_ALERT=${REACT_APP_EVENT_ALERT}
+      - SENTRY_DSN=${SENTRY_DSN}
+      - CHAOSGENIUS_ENTERPRISE_EDITION_KEY=${CHAOSGENIUS_ENTERPRISE_EDITION_KEY}
+      - CHAOSGENIUS_WEBAPP_URL=${CHAOSGENIUS_WEBAPP_URL}
+
+  chaosgenius-webapp:
+    container_name: chaosgenius-webapp
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile.prod
+    command: >
+      sh -c "npx react-inject-env set -d ./ &&
+             nginx -g 'daemon off;'"
+    volumes:
+      - ./frontend/nginx:/etc/nginx/conf.d/
+    ports:
+      - "8080:8080"
+    environment:
+      - REACT_APP_BASE_URL=${REACT_APP_BASE_URL}
+      - REACT_APP_DISABLE_TELEMETRY=${REACT_APP_DISABLE_TELEMETRY}
+      - REACT_APP_EVENT_ALERT=${REACT_APP_EVENT_ALERT}
+    depends_on:
+      - chaosgenius-server
+
+  chaosgenius-db:
+    container_name: chaosgenius-db
+    image: postgres:13
+    restart: always
```

Comment:
We don't have restart policies for the other services, so we should remove it from this one too

Reply:
@Samyak2  what do think about restart on-failure for db and redis ? or still default no will be better ?

Reply:
going for on-failure for all "dev" compose services